target/scripts/startup/setup-stack.sh

```diff
@@ -1263,3 +1263,22 @@ EOF
   supervisorctl reread
   supervisorctl update
 }
+
+function _setup_timezone
+{
+  _log 'debug' "Setting timezone to '${TZ}'"
+
+  export DEBCONF_NONINTERACTIVE_SEEN=true
+  local AREA ZONE DPKG_RECONFIGURE_PRESEED
+
+  AREA=$(cut -d '/' -f 1 <<< "${TZ}")
+  ZONE=$(cut -d '/' -f 2 <<< "${TZ}")
```

Comment:
```suggestion
  AREA=${TZ%%/*}
  ZONE=${TZ##*/}
```

maybe nicer to use pattern matching in parameter substitution then spawning subshell

Reply:
I like it :D

Reply:
What is this magic :mage_man: :monocle_face:

How you all remember all this syntax eludes me :joy:

Reply:
> What is this magic mage_man monocle_face

Parameter substitution is Bash's formidable way of adjusting the value than a variable contains. What @wernerfred proposed here is nice, because with the first line, we get everything _before_ the `/` character, and cut off everything behind it. The second line does the opposite, cutting off everything before the `/`, and keeping everything _after_ it. 

> How you all remember all this syntax eludes me joy

My go-to: <https://devhints.io/bash>